ipamd/ipamd.go

```diff
@@ -824,9 +824,9 @@ func (c *IPAMContext) nodeIPPoolTooLow() bool {
 	available := total - used
 	poolTooLow := available < c.maxIPsPerENI*c.warmENITarget || (c.warmENITarget == 0 && available == 0)
 	if poolTooLow {
-		log.Debugf("IP pool is too low: available (%d) < ENI target (%d) * addrsPerENI (%d)", available, c.warmENITarget, c.maxIPsPerENI)
+		log.Tracef("IP pool is too low: available (%d) < ENI target (%d) * addrsPerENI (%d)", available, c.warmENITarget, c.maxIPsPerENI)
```

Comment:
These seem to be informative logs, what's your motivation to move those to the lowest log level? Are you planning to not log your binary at this level?

Reply:
Yes, we log too much. Another option would be to change the default to be Info instead of Debug (see #257), but when testing stuff Trace would be helpful. I've updated the PR.

Reply:
So are we ok to log at `Info` level in production? Wouldn't it make it harder for us to debug issues after they've happened if we skip these logs?

Reply:
Yes, I'd like to keep the default at Debug level for now, but some of them are very verbose, that's why I'm lowering them to Trace for some of these.